gcalbot/db.sql

```diff
@@ -73,3 +73,21 @@ CREATE TABLE `invite` (
         ON DELETE CASCADE
     -- no foreign key to subscription, want to keep invites after unsubscribe so that users can still react to invites
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+DROP TABLE IF EXISTS `daily_schedule_subscription`;
+
+CREATE TABLE `daily_schedule_subscription` (
+    `keybase_username` varchar(128) NOT NULL,       -- kb username
+    `account_nickname` varchar(128) NOT NULL,       -- nickname of google account for kb user
+    `calendar_id` varchar(128) NOT NULL,            -- google calendar id that this subscription is for
+    `keybase_conv_id` char(64) NOT NULL,            -- channel that is subscribed to notifications
+    `timezone` varchar(128) NOT NULL,               -- timezone that this subscription should respect
+    `days_to_send` ENUM ('everyday', 'monday through friday', 'sunday through thursday'), -- days of the week to send notifications
+    `schedule_to_send` ENUM ('today', 'tomorrow'),  -- schedule to send
+    `notification_duration` int(11) NOT NULL,       -- minutes after beginning of UTC day before notification should be sent
```

Comment:
why not a sql time field?

Reply:
I suppose I could do that, I will still convert to a duration on the go side though if you think that's alright.

Reply:
yeah that's fine